.github/workflows/run_benches.yml

```diff
@@ -47,42 +46,32 @@ jobs:
       - name: Install rust toolchain
         uses: dtolnay/rust-toolchain@stable
 
-      - name: Install critcmp
```

Comment:
I would prefer to keep critcmp as second option for now.

Reply:
Sounds great!
I've added `critcmp` back and it is now running in tandem with Bencher 😃
Example run here: https://github.com/epompeii/diesel/actions/runs/7767012585#summary-21183420265